src/lib/Rattletrap/Console/Main.hs

```diff
@@ -132,18 +129,7 @@ versionMain = do
   putStrLn Version.string
 
 schemaMain :: Config.Config -> IO ()
-schemaMain config = do
-  let
-    json = Aeson.encodePretty'
-      Aeson.defConfig
-        { Aeson.confCompare = compare
-        , Aeson.confIndent = Aeson.Tab
-        , Aeson.confTrailingNewline = True
-        }
-      schema
-  case Config.output config of
-    Nothing -> LazyByteString.putStr json
-    Just file -> LazyByteString.writeFile file json
+schemaMain config = putOutput config $ encodeJson config schema
```

Comment:
As a result of this change, you can generate a compact schema with `rattletrap --schema --compact`.